src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -192,14 +190,19 @@
         /** Optionally override http://guides.dataverse.org . */
         GuidesBaseUrl,
 
-        CVMConf,
+        CVocConf,
 
         /**
          * A link to an installation of https://github.com/IQSS/miniverse or
          * some other metrics app.
          */
         MetricsUrl,
-        
+
+        /**
+         * Key for Setting of Signposting
+         */
+        SignpostingConf,
```

Comment:
Strange, I expected this in the 'signposting' feature branch, somehow it now ia also here?

Reply:
According to the annotations you added those lines today, and I don't see it being used.

Reply:
done.